examples/ngsi_v2/e13_ngsi_v2_secure_fiware_headers.py

```diff
@@ -0,0 +1,111 @@
+import os
+
+import requests
+import urllib3
+
+from filip.clients.ngsi_v2 import ContextBrokerClient
+from filip.config import settings
+from filip.models.base import FiwareHeaderSecure
+
+urllib3.disable_warnings()
+session = requests.Session()
+CB_URL = settings.CB_URL
+SERVICE = 'filip'
+SERVICE_PATH = '/'
+CLIENT_ID = 'client_id'
+CLIENT_SECRET = 'client_secret'
+KEYCLOAK_HOST = 'https://keycloak.example.com'
```

Comment:
@sbanoeon could you generally add some hints for example here, `TODO: adapt it to your authentication endpoint`